### PR TITLE
[CP-1491] Fix templates selection

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1491]

**Description**
Adding selection functionality to templates

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1491]: https://appnroll.atlassian.net/browse/CP-1491?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/templates/components/templates/templates.component.tsx [hunk 1]
```diff
@@ -54,8 +59,8 @@ export const Templates: FunctionComponent<TemplatesProps> = ({
   const panelButtonDisabled =
     templateFormOpen || states.creating || states.updating
 
-  const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
-    useTableSelect<Template>(templates)
+  // const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
+  //   useTableSelect<Template>(templates)
```
Comment (wojcik92michal) [inline packages/app/src/templates/components/templates/templates.component.tsx:63]:
```suggestion
```

**Review by wojcik92michal — DISMISSED** [review]

**Review by wercix — COMMENTED** [review]
Templates checkboxes are still affected by data sync. Check out the reproduction path for templates.

File: packages/app/src/templates/components/templates/templates.component.tsx (hunk 1 above)
Comment (igorbezsmertnyi) [reply packages/app/src/templates/components/templates/templates.component.tsx]: https://github.com/mudita/mudita-center/pull/885/commits/fc2a17b4976f64a52756931bcb2b64f18911e1e1

**Comment by igorbezsmertnyi** [comment]
> Templates checkboxes are still affected by data sync. Check out the reproduction path for templates.

@wercix I should disagree with you. It's works perfectly

https://user-images.githubusercontent.com/21984930/183051592-a7bbc52c-8278-463e-a9b5-f2d42aa699a9.mov

**Comment by dkarski** [comment]
> > Templates checkboxes are still affected by data sync. Check out the reproduction path for templates.
> 
> @wercix I should disagree with you. It's works perfectly
> 
>  Screen.Recording.2022-08-05.at.11.43.46.AM.mov

here you are after sync 😉

**Comment by wercix** [comment]
> > Templates checkboxes are still affected by data sync. Check out the reproduction path for templates.
> 
> @wercix I should disagree with you. It's works perfectly
> 
>  Screen.Recording.2022-08-05.at.11.43.46.AM.mov

It is not the reproduction path on your film. Data sync should not be finished before selecting checkbox

https://user-images.githubusercontent.com/15948014/183053613-92f258c6-b61f-4a1a-b292-32d876aa3ebf.mov

**Comment by igorbezsmertnyi** [comment]
@wercix @dkarski 
https://user-images.githubusercontent.com/21984930/183053934-312e4849-87a5-4ef0-913e-8c8c1017d093.mov

**Comment by dkarski** [comment]
@wojcik92michal could please re-test this issue? 🙏

**Comment by wojcik92michal** [comment]
In my case it is working fine. Tested on dev env, MuditaOS 1.3.0-rc.8, MacOS 12.4

https://user-images.githubusercontent.com/41268731/183073667-5f473525-6672-4c2f-8500-60e966949d68.mov

**Review by dkarski — APPROVED** [review]
I can't too 🤔 

https://user-images.githubusercontent.com/17147149/183077624-c8a94767-c03d-4269-9d13-829d5e927139.mov

Tested on dev/alpha/production env, MuditaOS 1.3.0, MacOS 12.4

**Review by wercix — APPROVED** [review]
I checked it one more time and it works

**Pull request merged**